src/expression/embeddedDocs/function/probability/lgamma.js

```diff
@@ -0,0 +1,13 @@
+export const lgammaDocs = {
+  name: 'lgamma',
+  category: 'Probability',
+  syntax: [
+    'lgamma(n)'
+  ],
+  description: 'Compute the log gamma function of a value using Lanczos approximation.',
+  examples: [
```

Comment:
I thought you said you're no longer using Lanczos? Please correct if so. Also, the examples should include a complex argument.

Reply:
hmmm actually I'm still using Lanczos for numbers, but Stirling series for complex numbers.

Reply:
OK, you can mention that, or mention neither in the embedded docs, only mention that kind of detail in the full jsdoc-style docs that turn into the web docs.

Reply:
Yeah I changed it to mention both.